datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SetRangerCloudIdentityMappingRequest.java

```diff
@@ -42,4 +43,22 @@ public String toString() {
                 ", azureGroupMapping=" + azureGroupMapping +
                 '}';
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        SetRangerCloudIdentityMappingRequest that = (SetRangerCloudIdentityMappingRequest) o;
+        return azureUserMapping.equals(that.azureUserMapping) &&
```

Comment:
Should we use Objects.equals for this to be safe? The current code assumes that the fields are not null. I see the fields are annotated @NotNull, but that just affects validation.

Reply:
True, I've updated the equals to be null safe.